src/widgets/DataTable/Cell.js

```diff
@@ -13,15 +13,16 @@ import { getAdjustedStyle } from './utilities';
  * @param {Object}        textStyle  Style object for the inner Text
  * @param {Number}        width      optional flex property to inject into styles.
  * @param {Bool}          isLastCell Indicator for if this cell is the last
+ * @param {Number}        maxLiens   Maximum number of lines for the Text component
  *                                   in a row. Removing the borderRight if true.
  */
-const Cell = React.memo(({ value, textStyle, viewStyle, width, isLastCell, debug }) => {
+const Cell = React.memo(({ value, textStyle, viewStyle, width, isLastCell, maxLines, debug }) => {
   if (debug) console.log(`- Cell: ${value}`);
   const internalViewStyle = getAdjustedStyle(viewStyle, width, isLastCell);
 
   return (
     <View style={internalViewStyle}>
-      <Text ellipsizeMode="tail" numberOfLines={1} style={textStyle}>
+      <Text ellipsizeMode="tail" numberOfLines={maxLines} style={textStyle}>
```

Comment:
Just my opinion, think its nice to use consistent props with the underlying component. E.g. `numberOfLines` instead of `maxLines`. I think unless the underlying component is being used in a very different way, it makes sense to model components consistently with the conventions used by the built-in components?

Reply:
352447b432b5fde3924d06546c5fbe067c662eea